uktrade/fido
# Feature/add row

This feature adds the ability to add a row to a forecast. Currently, as there is no edit function, it's not possible to edit the row once added.

In order to test this feature, you will need to start with a blank database, migrate and then run:
```
docker-compose run fido python manage.py create_stub_data All
```

To test:

- Go to the edit page.
- Click the "Add a new row" button
- Select relevant values from the drops downs on the add a row page
- Click "Add entry" and check that a new row is added to the edit table

Nb. This feature also includes, breadcrumbs and a revision of the main SCSS file to remove redundant styles.
